modules/light-clients/07-tendermint/types/update_test.go

```diff
@@ -367,6 +167,228 @@ func (suite *TendermintTestSuite) TestCheckHeaderAndUpdateState() {
 	}
 }
 
+func (suite *TendermintTestSuite) TestVerifyHeader() {
+	var (
+		path   *ibctesting.Path
+		header *ibctmtypes.Header
+	)
+
+	// Setup different validators and signers for testing different types of updates
+	altPrivVal := ibctestingmock.NewPV()
+	altPubKey, err := altPrivVal.GetPubKey()
+	suite.Require().NoError(err)
+
+	revisionHeight := int64(height.RevisionHeight)
+
+	// create modified heights to use for test-cases
+	altVal := tmtypes.NewValidator(altPubKey, revisionHeight)
```

Comment:
Isn't the second argument supposed to be voting power?

Reply:
yes. I'd remove `revisionHeight` and replace with `100`

Reply:
good catch. This was leftover from the previous test